app/javascript/components/city-name.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+import styled from 'styled-components';
+import { WhiteSubTitle, WhiteTitleLarge } from '@components/typography';
+import { LoadingMessage, ErrorMessage } from '@messages/default-messages';
+
+const getCityName = gql`
+    {
+        primaryLocation {
+            cityName
+        }
+    }
+`;
+
+const Column = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-top: 100px;
+`;
+// const Column = styled.div`
+//     font-size: ${fontSizes.xlarge};
+//     font-family: ${fonts.extended};
+//     font-weight: ${weights.regular};
+//     position: relative;
+//     top: 8px;
+// `;
+
+export const City = () => (
+    <Query query={getCityName}>
```

Comment:
Ah! I forgot side panel wasn't using the newer graphql fragment pattern. Out of scope here so I'll make a new ticket.